chroma-manager/tests/framework/integration/installation_and_upgrade/cluster_setup

```diff
@@ -30,13 +30,27 @@ if [ ${PIPESTATUS[0]} != 0 ]; then
 fi
 
 # Install and setup integration tests on integration test runner
-scp $CLUSTER_CONFIG root@$TEST_RUNNER:/root/cluster_cfg.json
+if ! $JENKINS; then
+    CMIT=$(ls chroma-manager/dist/chroma-manager-integration-tests-*.x86_64.rpm)
+fi
+scp $CMIT $CLUSTER_CONFIG root@$TEST_RUNNER:/root/
```

Comment:
Is it possible that CMIT isn't defined? I don't see a reference above the if statement.

Reply:
If undefined, it expands to nothing so it's a NOOP in the `scp`.

Additionally below, at line 41, again in the undefined case it is just given a local value.